aria/modeling/utils.py

```diff
@@ -51,74 +53,95 @@ def service_template(self):
         return self.container.service_template
 
 
-def create_inputs(inputs, template_inputs):
+def merge_parameter_values(parameter_values, declared_parameters, forbidden_names=None):
```

Comment:
what was the purpose of the changes to this method? im not sure i understand the benefit and it seems less readable now to me